.github/workflows/ci.yml

```diff
@@ -0,0 +1,46 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - "*"
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v1
+        with:
+          fetch-depth: 100
+
+      - name: Install and run Luacheck
+        uses: nebularg/actions-luacheck@v1
```

Comment:
There is also a `luacheck@v2` version available already which we should use.

Moreover will this also run luacheck and not just install it? 🤔 For my other addon I added:

```yml
      - uses: leafo/gh-actions-lua@v8.0.0
        with:
          luaVersion: "5.1"

      - uses: leafo/gh-actions-luarocks@v4.0.0

      - name: build
        run: |
          luarocks install luacheck
          luacheck .
```

to install lua + luarocks and run the luacheck command.

Reply:
Yeh it does both installing and running.